src/components/engine/create-engine-instance.tsx

```diff
@@ -39,24 +45,28 @@ export const CreateEngineInstanceButton = ({
   const trackEvent = useTrack();
   const toast = useToast();
   const accountQuery = useAccount();
+  const [engineTier, setEngineTier] = useState<EngineTier>("STARTER");
 
-  const requestCloudHostedEngine = async () => {
+  const addCloudHostedEngine = async () => {
     trackEvent({
       category: "engine",
       action: "click",
       label: "clicked-cloud-hosted",
+      tier: engineTier,
     });
 
     try {
       const res = await fetch(
-        `${THIRDWEB_API_HOST}/v1/engine/add-cloud-hosted`,
+        `${THIRDWEB_API_HOST}/v1/engine/add-cloud-hosted-XXX`,
```

Comment:
WIP or is this actually the endpoint?

Reply:
Whoops WIP to not actually hit this flow

Reply:
fixed this